website/versioned_docs/version-2.0.0/user-guides/upgrade.md

```diff
@@ -0,0 +1,28 @@
+---
+id: upgrade
+title: Upgrade Litmus ChaosCenter to Latest Version
+sidebar_label: Upgrade
+---
+
+---
+
+To upgrade Litmus ChaosCenter from 2.0.0-Beta9 to 2.0.0, you can follow these steps:
+
+1. ## Control Plane Upgrade
+    With this approach, the ChaosCenter components will get upgrade via helm. This step updates the frontend, server and mongo image. This can be done using the following commands: 
+
+    ```
+    kubectl set image
+    ```
+
+    ```
+    kubectl set env VERSION=“2.0.0”
+    ```
+
+2. ## Agent Plane Upgrade
+    With this approach, all the ChaosAgents connected with ChaosCenter should be upgraded manually with [Litmusctl](../litmusctl/installation).
```

Comment:
I suppose we need a `disconnect ` operation from the chaos center, and following that a removal of the other deployments and cm (disconnect doesn't remove other components by itself today)

This should be followed by the litmusctl connect again if I am not wrong @gdsoumya ?

Reply:
Sure, will add the `disconnect` step.

Reply:
@ksatchit disconnect from the portal is like deleting subscriber and agent-config cm. We can do it manually or portal.